pypot/robot/config.py

```diff
@@ -134,6 +134,15 @@ def dxl_io_from_confignode(config, c_params, ids, strict):
         port = pypot.dynamixel.find_port(ids, strict)
         logger.info('Found port {} for ids {}'.format(port, ids))
 
+    sync_read = c_params['sync_read']
+
+    if sync_read == 'auto':
+        # USB VID:PID=0403:6001 is specific to USB2DYNAMIXEL
```

Comment:
I think we should do the opposite. If it's a USB2AX sets sync_read to True.
If we use another controller, we don't know if it will support sync_read or not.

You should also check if we are using protocol v1 or v2. The USB2Dynamixel can handle sync_read when used with v2 motors (such as XL-320).

Finally, a one-liner would be some much more pythonic ^^